platforms/documentation/docs/src/docs/userguide/dep-man/dependency-management/component_metadata_rules.adoc

```diff
@@ -78,153 +72,130 @@ include::sample[dir="snippets/dependencyManagement/customizingResolution-metadat
+However, the `pom` file lacks information about these features and their corresponding dependencies.
+This can be represented in Gradle Module Metadata through variants and <<component_capabilities.adoc#declaring-component-capabilities,capabilities>>, which we can add via a component metadata rule.
 
-.Rule to add optional feature to Jaxen metadata
 ====
 include::sample[dir="snippets/dependencyManagement/customizingResolution-metadataRule/kotlin",files="build.gradle.kts[tags=jaxen-rule-2]"]
 include::sample[dir="snippets/dependencyManagement/customizingResolution-metadataRule/groovy",files="build.gradle[tags=jaxen-rule-2]"]
 ====
 
-Here, we first use the `addVariant(name, baseVariant)` method to create an additional variant, which we identify as _feature variant_ by defining a new capability _jaxen-dom4j_ to represent the optional dom4j integration feature of Jaxen.
-This works similar to <<feature_variants.adoc#sec::declare_feature_variants,defining optional feature variants>> in build scripts.
-We then use one of the `add` methods for adding dependencies to define which dependencies this optional feature needs.
+In this example, we create a new variant called `runtime-dom4j` using the `addVariant(name, baseVariant)` method.
+This variant represents an optional feature, defined by the capability `jaxen-dom4j`.
+We then add the required dependency `dom4j:dom4j:1.6.1` to this feature.
 
-In the build script, we can then add a <<feature_variants.adoc#sec::consuming_feature_variants, dependency to the optional feature>> and Gradle will use the enriched metadata to discover the correct transitive dependencies.
-
-.Applying and utilising rules for Jaxen metadata
 ====
 include::sample[dir="snippets/dependencyManagement/customizingResolution-metadataRule/kotlin",files="build.gradle.kts[tags=jaxen-dependencies]"]
 include::sample[dir="snippets/dependencyManagement/customizingResolution-metadataRule/groovy",files="build.gradle[tags=jaxen-dependencies]"]
 ====
 
-== Making variants published as classified jars explicit ==
+By applying these rules, Gradle uses the enriched metadata to correctly resolve the optional dependencies when the `jaxen-dom4j` feature is required.
+
+== Making variants published as classified jars explicit
 
-While in the previous example, all variants, "main variants" and optional features, were packaged in one jar file, it is common to publish certain variants as separate files.
-In particular, when the variants are mutual exclusive — i.e. they are *not* feature variants, but different variants offering alternative choices.
-One example *all* pom-based libraries already have are the _runtime_ and _compile_ variants, where Gradle can choose only one depending on the task at hand.
-Another of such alternatives discovered often in the Java ecosystems are jars targeting different Java versions.
+In some cases, variants are published as separate files rather than being packaged together in a single jar.
```

Comment:
🤔 I'd change this.  It's much more common than not for each variant to supply different artifacts.  That's usually part of the purpose for making different variants.

Reply:
Yup, that was old docs. Updated.